File: Store/src/Sales.Domain/Model/Orders/OrderItem.cs
using System;

namespace Sales.Domain.Model.Orders
{
    public class OrderItem
    {
        public long ProductId { get; private set; }
        public long Quantity { get; private set; }
        public long EachPrice { get; private set; }
        public OrderItem(long productId, long quantity, long eachPrice)
        {
            ProductId = productId;
            Quantity = quantity;
            EachPrice = eachPrice;
        }
        public long TotalPrice()
        {
            return 0;
        }
    }
}

File: src/main/java/store/sales/domain/model/orders/OrderItem.java
package store.sales.domain.model.orders;

public class OrderItem {
    private final int productId;
    private final int quantity;
    private final int eachPrice;

    public OrderItem(int productId, int quantity, int eachPrice) {
        this.productId = productId;
        this.quantity = quantity;
        this.eachPrice = eachPrice;
    }

    public int totalPrice() {
        return eachPrice * quantity;
    }

    public int getProductId() {
        return productId;
    }

    public int getQuantity() {
        return quantity;
    }

    public int getEachPrice() {
        return eachPrice;
    }
}
